app/src/main/java/com/practicum/pl_maker/Track.kt

```diff
@@ -0,0 +1,9 @@
+package com.practicum.pl_maker
+
+class Track(
+    val trackName: String,
+    val musicianName: String,
+    val trackTime: String,
+    val artworkUrl100: String
+) {
+}
```

Comment:
```suggestion
)
```
Лучше сделать data class
Тело в data class прописывать не стоит, если оно все равно пустое